index.html

```diff
@@ -0,0 +1,223 @@
+<link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+  <main class="container">
+    <section class="introduction">
+      <header class="main-header">
+        <!-- flex-container -->
+        <div class="toolbar"> 
+          <!-- flex-items -->
+            <div class="three time"><img src="./images/Time.png" alt=""></div>
+            <nav>
+              <div class="three"><img src="./images/Cellular.png" alt=""></div>
+              <div class="three"><img src="./images/Wifi.png" alt=""></div>
+              <div class="three"><img src="./images/Battery.png" alt=""></div>
+            </nav>
+      </div>
+      <div class="logo">
+        <div><img src="./images/Logo.png" alt=""></div>
+        <div><img src="./images/Union.png" alt=""></div>
+      </div>
+      </header>
+      <div class="images">
+        <div class="header-images">
+          <div id="elipse11">
+              <img src="./images/Ellipse11.png" alt="">
+          </div>
+          <div id="elipse15">
+              <img src="./images/Ellipse15.png" alt="">
+          </div>
+          <div id="rectangle7">
+              <img src="./images/Rectangle7.png" alt="">
+          </div>
+          <div id="elipse3">
+              <img src="./images/Ellipse3.png" alt="">
+          </div>
+          <div id="rectangle4">
+              <img src="./images/Rectangle4.png" alt="">
+          </div>
+          <div id="elipse8">
+              <img src="./images/Ellipse8.png" alt="">
+          </div>
+          <div id="elipse12">
+            <img src="./images/Ellipse12.png" alt="">
+          </div>
+          <div id="elipse13">
+            <img src="./images/Ellipse13.png" alt="">
+          </div>
+          <div class="elipse8">
+            <img src="./images/Ellipse8.png" alt="">
+          </div>
+          <div class="elipse4">
+            <img src="./images/Ellipse4.png" alt="">
+          </div>
+          <div class="rectangle5">
+            <img src="./images/Rectangle5.png" alt="">
+          </div>
+        </div>
+      </div>
+      <div class="name">
+        <p>I’m Benjamin
+          Glad to see you!</p>
+      </div>
+      <p class="details">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience!
+         If you like what you see and have a project you need coded,
+          don’t hestiate to contact me.</p>
+          <div class="lets-connect">
+            <p class="connect">LET’S CONNECT</p>
+          </div>
+          <div class="media-icons">
+              <a href=""><img src="./images/twitter.png" alt=""></a>
+              <a href=""><img src="./images/Linkedin.png" alt=""></a>
+               <a href=""><img src="./images/mail.png" alt=""></a>
+               <a href=""><img src="./images/github.png" alt=""></a>
+              <a href=""><img src="./images/peace.png" alt=""></a>
+          </div>
+    </section>
+    <article class="workcards">
```

Comment:
Kindly, by convention the ARTICLE tag is used for repetitive objects, I recommend you use the MAIN semantic tag